test/src/integration/integration_scalardl_test.go

```diff
@@ -115,3 +122,26 @@ func getExternalIP(t *testing.T) string {
 
 	return output
 }
+
+func checkTCPConnect(t *testing.T, host string) string {
```

Comment:
I think it's a bit more readable to name it something like `isReachable` and return a boolean value.
```suggestion
func isReachable(t *testing.T, host string) string {
```

Reply:
fixed in 3d9625e